lib/transaction.js

```diff
@@ -61,8 +61,25 @@ Transaction.ISOLATION_LEVELS = {
  *   lock: t1.LOCK.SHARE,
  *   lock: t1.LOCK.KEY_SHARE, // Postgres 9.3+ only
  *   lock: t1.LOCK.NO_KEY_UPDATE // Postgres 9.3+ only
- * })
+ * });
  * ```
+ * 
+ * Postgres also supports specific locks while eager loading by using OF.
+ * 
+ * ```js
+ * UserModel.findAll({
+ *   where: ...,
+ *   include: [TaskModel, ...]
+ * }, {
+ *   transaction: t1,
```

Comment:
This is moved to the main object, queryOptions is gone

(well it isn't currently, but it will be in 3.0)